src/test/java/edu/cornell/kfs/pdp/batch/service/impl/PayeeACHAccountExtractServiceImplTest.java

```diff
@@ -744,65 +741,87 @@ public void markRowAsInvalid() {
     }
     
     @Test
     public void testCleanPayeeACHAccountExtractDetailWithEmptyString() {
-        validateCleanPayeeACHAccountExtractDetail(StringUtils.EMPTY, StringUtils.EMPTY);
+        validateCleanPayeeACHAccountExtractDetailBankAccountNumber(StringUtils.EMPTY, StringUtils.EMPTY);
     }
     
     @Test
     public void testCleanPayeeACHAccountExtractDetailWithSpaces() {
-        validateCleanPayeeACHAccountExtractDetail(" 1 23  4 5 6 ", "123456");
+        validateCleanPayeeACHAccountExtractDetailBankAccountNumber(" 1 23  4 5 6 ", "123456");
     }
     
     @Test
     public void testCleanPayeeACHAccountExtractDetailWithSpacesAndDashes() {
-        validateCleanPayeeACHAccountExtractDetail(" 1 2----3  4 - 5 6-7 ", "1234567");
+        validateCleanPayeeACHAccountExtractDetailBankAccountNumber(" 1 2----3  4 - 5 6-7 ", "1234567");
     }
     
     @Test
     public void testCleanPayeeACHAccountExtractDetailWithSpacesAndDashesAndLetters() {
-        validateCleanPayeeACHAccountExtractDetail("A 1 2----3  4 - 5 6-7 B ", "A1234567B");
+        validateCleanPayeeACHAccountExtractDetailBankAccountNumber("A 1 2----3  4 - 5 6-7 B ", "A1234567B");
     }
     
     @Test
     public void testCleanPayeeACHAccountExtractDetailWithJustSpaces() {
-        validateCleanPayeeACHAccountExtractDetail("   ", StringUtils.EMPTY);
+        validateCleanPayeeACHAccountExtractDetailBankAccountNumber("   ", StringUtils.EMPTY);
     }
-    
-    private void validateCleanPayeeACHAccountExtractDetail(String bankAccountNumber, String expectedCleanedBankAccountNumber) {
+
+    @Test
+    public void testBlankBankName() {
+        validateCleanPayeeACHAccountExtractDetailBankName("   ", StringUtils.EMPTY);
+    }
+
+    @Test
+    public void testStandardBankName() {
+        validateCleanPayeeACHAccountExtractDetailBankName("Example Bank", "Example Bank");
+    }
+
+    @Test
+    public void testLongBankName() {
+        validateCleanPayeeACHAccountExtractDetailBankName("Tompkins Community Bank and Sons forty characters test", "Tompkins Community Bank and Sons forty c");
+    }
+
+    private void validateCleanPayeeACHAccountExtractDetailBankAccountNumber(String bankAccountNumber, String expectedCleanedBankAccountNumber) {
         PayeeACHAccountExtractDetail detail = new PayeeACHAccountExtractDetail();
         detail.setBankAccountNumber(bankAccountNumber);
         payeeACHAccountExtractService.cleanPayeeACHAccountExtractDetail(detail);
         
         assertEquals(expectedCleanedBankAccountNumber, detail.getBankAccountNumber());
     }
+    private void validateCleanPayeeACHAccountExtractDetailBankName(String bankName, String expectedCleanedBankName) {
+        PayeeACHAccountExtractDetail detail = new PayeeACHAccountExtractDetail();
+        detail.setBankName(bankName);
+        payeeACHAccountExtractService.cleanPayeeACHAccountExtractDetail(detail);
+
+        assertEquals(detail.getBankName(), expectedCleanedBankName);

```

Comment:
In this JUnit test assertion, the first argument should be the expected value and the second argument should be the actual value.